HelpAssistHer/help-assist-her
# Add a filter so only verified results are displayed

## Description
Update the queries so that only the verified results are returned upon searching by address.

## Test Plan
1. Verify the name, address, website, and phone number for a pregnancy center (the verified date needs to be after Oct 31, 2018)
1. Note the zip code so you can search for that particular PRC
1. Go to http://localhost:4000/mini-app and type in the zip code
1. Make sure only that 1 result that was verified after Oct 31 and has the name, address, phone, and website verified shows up
